modules/learn/pages/services-and-indexes/services/backup-service.adoc

```diff
@@ -250,3 +247,4 @@ You may want to experiment with different thread settings to find a balance betw
 * See xref:rest-api:backup-rest-api.adoc[Backup Service API] for information about using the Backup Service from the REST API.
 * To learn about the port numbers the Backup Service uses, see xref:install:install-ports.adoc[Couchbase Server Ports].
 * For a list of Backup Service audit events, see xref:audit-event-reference:audit-event-reference.adoc[Audit Event Reference].
+* For information about adding or removing a Backup Service on a node, see xref:manage:manage-nodes/modify-services-and-rebalance.adoc[Modify Services and Rebalance].
```

Comment:
Same thing -- instead of "For information about adding or removing a Backup Service on a node," -- clarify with "For information about adding or removing a Backup Service on an existing node,".

Side note:  People will be happy to see that blurb about "on an existing node" since that is something they've been asking for -- to be able to add the Backup Service to an existing node of a cluster instead of having to configure the Backup Service on a new node, then, join/add the new node to a cluster.